src/locales/zh_CN/modifier-type.ts

```diff
@@ -215,6 +215,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "TOXIC_ORB": { name: "剧毒宝珠", description: "触碰后会放出毒的神奇宝珠。携带后，在战斗时会变成剧毒状态。" },
     "FLAME_ORB": { name: "火焰宝珠", description: "触碰后会放出热量的神奇宝珠。携带后，在战斗时会变成灼伤状态。" },
+    "ASSAULT_VEST": { name: "Assault Vest", description: "虽然携带后特防会提高50%，、\n但会无法使出变化招式。(除了抢先一步)" },
```

Comment:
```suggestion
    "ASSAULT_VEST": { name: "突击背心", description: "虽然携带后特防会提高50%，、\n但会无法使出变化招式。(除了抢先一步)" },
```

I forgot the item name sorry